html/desserts.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Berkshire+Swash&display=swap"
+      rel="stylesheet"
+    />
+    <title>My Asian Kitchen</title>
+
+    <link rel="stylesheet" href="../styles.css" />
+  </head>
+  <div id="home">
```

Comment:
I think you should use a class here instead as ids are supposed to be unique. Here you reuse the home id quite a lot.